docs/guide/plot_what.rst

```diff
@@ -0,0 +1,4 @@
+.. _plot_what:
+
+How about ENa-Vm, i.e. the driving force for INa? Maybe normalize this with respect to the driving force at t = 0?
+
```

Comment:
why does this page exist? was there supposed to be more?

Reply:
Maybe figure out where it's linked from and put that text there (and remove the page)... or just remove the page, whichever makes more sense

Reply:
we decided this was fine in the context of the page